### DISCUSSION TITLE
GitHub sending emails concerning Heroku and Travis CI OAuth attacks

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#126] [Show and tell]
This is an announcement to say that GitHub has found an OAuth malicious attacks based on the Heroku and Travis CI service. Certain people (including me) might be affected from it.

I just recieved this email recently : 

```
Hi LaneSh4d0w,

We're writing to let you know that the names of the organizations your GitHub account is a member of, if any, were likely viewed as part of an attack campaign against GitHub customers. Unless you have received a separate notification from GitHub indicating otherwise, we do NOT have evidence to suggest that the attacker used your account to view other private information, such as repository names or content.

We've shared details of this campaign on our blog:

https://github.blog/2022-04-15-security-alert-stolen-oauth-user-tokens/

GitHub provides a platform that third-parties can integrate with by building various apps. These apps, and the secrets associated with them, are in control of the third-parties that build, own, and maintain the apps. Loss of control of these secrets can result in unauthorized parties using them to access the GitHub repositories for which they are authorized. This attack campaign did not result from a compromise of GitHub.com, but rather relates to the compromise of these secrets held by Heroku and Travis CI.

If you have questions or need assistance regarding affected OAuth applications maintained by Heroku, please contact Heroku Support (https://help.heroku.com/). Updates can be found on their status site: https://status.heroku.com/incidents/2413

If you have questions or need assistance regarding affected OAuth applications maintained by Travis CI, please reach out to compliance@travis-ci.com.
```

EDIT : Upon further review of my account, the incriminated tokens were taken off days before the attack, I should be safe.

**This is a public announcement from a member of the iptv-org team. I, being potentially hit by the attack, will revoke my Travis CI & Heroku permissions immediately. I am sorry if certain "weird" actions were done inside these repositories, those weren't mine.**

**Comment by Mush-A** [comment]
I got this too. The funny thing is I got an email from Facebook as well and the email ended in the same fashion:

"Thanks,
GitHub Security"

and 

"Thanks
Facebook Security"

Is this email legit? I do not think so.

**Reply by BellezaEmporium** [reply]
Mine was, since it came from a trusted GitHub email address. For Facebook, it might be possible too since they use integrations, but I can't tell if they use OAuth.
